plugin/trino-bigquery/pom.xml

```diff
@@ -279,6 +279,12 @@
         <dependency>
             <groupId>org.apache.arrow</groupId>
             <artifactId>arrow-vector</artifactId>
+            <exclusions>
+                <exclusion>
```

Comment:
a comment about what it conflicts with helps when updating versions in future or removing exclusions in future